ARM_DisassemblerExplorer/src/parser/Instruction.cpp

```diff
@@ -0,0 +1,28 @@
+/**
+ * @addtogroup  Parser
+ * @{
+ * @file        Instruction.cpp
+ * @author      Pascal-Emmanuel Lachance
+ * @p           https://www.github.com/Raesangur
+ * @date        2020/07/24  -  20/02
+ */
+/*************************************************************************************************/
+/* Includes ------------------------------------------------------------------------------------ */
+#include "Instruction.h"
+
+
+namespace Parser
+{
+
+
+/*************************************************************************************************/
+/* Member functions definitions ---------------------------------------------------------------- */
+Instruction::~Instruction() = default;
```

Comment:
this can be done in the header file, I don't see why we'd need a cpp file just for that

Reply:
the header file declares it as pure virtual, but I encounter linker problems when only declaring a destructor as pure virtual, so I give it a later definition that only the linker will see.
It's a lazy fix, but it won't compile until I add another pure virtual method to the class.
Unless I'm missing something? 🤔

Reply:
afaik, you're not supposed to declare a dtor as `= 0` in C++, might be wrong.
If I'm correct, it's due to the need for C++ to know what to do with the memory upon destruction of the object, even if it's a pure virtual class that will never be instantiated

Reply:
oh that's why it doesn't work 😅